src/Native/format-code.sh

```diff
@@ -8,7 +8,7 @@ elif which "clang-format" > /dev/null 2>&1 ; then
    export CF="$(which clang-format)"
 else
    echo "Unable to find clang-format"
-   exit 1
+   export CF="/usr/local/llvm37/bin/clang-format"
```

Comment:
Removed. Now the format change is only scoped to the document i have changed in this PR. :)